### DISCUSSION TITLE
Homepage link do not appear to be working

### DISCUSSION THREAD

**Discussion by kuroshii-akr** [babel/babel#14894] [General]
Hello.
The links on npm page and package.json of two helpers do not appear to be working. Can you possibly fix this situation?

- @babel/helper-environment-visitor
  npm page https://www.npmjs.com/package/@babel/helper-environment-visitor
  Homepage link is https://babel.dev/docs/en/next/babel-helper-environment-visitor .
  This page redirects to https://babel.dev/docs/en/babel-helper-environment-visitor, and this page is 404.
  Homepage link seem reflected in node_modules/@babel/helper-environment-visitor/package.json .

- @babel/helper-function-name
  npm page https://www.npmjs.com/package/@babel/helper-function-name .
  Homepage link is https://babel.dev/docs/en/next/babel-helper-function-name .
  This page redirects to https://babel.dev/docs/en/babel-helper-function-name, and this page display only "TODO".
  Homepage link seem reflected in node_modules/@babel/helper-function-name/package.json .

**Comment by JLHwung** [comment]
`@babel/helper-*` are internal Babel helpers so they are not very well documented. Are you working on a Babel plugin?

**Reply by kuroshii-akr** [reply]
> @babel/helper-* is an internal helper for Babel, so it is not well documented.

I see.

I'm not developing a plugin, so I don't have a development problem.

I use Babel, so I have the MIT license displayed.
I am quoting the homepage in the package.json to display the license.
So I hope it is a valid link.

How about changing the link to "https://babel.dev/"?
I know it is hard to modify the babel/helper-* documentation.
